seqr/views/apis/family_api_tests.py

```diff
@@ -89,17 +88,22 @@ def test_family_page_data(self, mock_analyst_group):
         # Test analyst users have internal fields returned
         self.login_analyst_user()
         response = self.client.get(url)
-        self.assertEqual(response.status_code, 403)
-
-        mock_analyst_group.__bool__.return_value = True
-        mock_analyst_group.resolve_expression.return_value = 'analysts'
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        family_fields.update(CASE_REVIEW_FAMILY_FIELDS)
+        internal_family_fields = deepcopy(family_fields)
+        internal_family_fields.update(INTERNAL_FAMILY_FIELDS)
+        individual_fields.update(CASE_REVIEW_INDIVIDUAL_FIELDS)
+        internal_individual_fields = deepcopy(individual_fields)
+        internal_individual_fields.update(INTERNAL_INDIVIDUAL_FIELDS)
+        self.assertSetEqual(set(response_json['familiesByGuid'][FAMILY_GUID].keys()), internal_family_fields)
+        self.assertSetEqual(set(next(iter(response_json['individualsByGuid'].values())).keys()), internal_individual_fields)
+
+        self.mock_analyst_group.__str__.return_value = ''
```

Comment:
This line of code makes the `analyst_user` a non-analyst user. Right? Adding a comment or logging in with a different test user is better.

Reply:
Not at all. This is testing seqr's behavior when no `ANALYST_USER_GROUP` is configured, which is the default setting for local installs. It is is crucial to test that seqr works as expected when the group is entirely disabled, which is very different than testing the behavior for a non-analyst user in a seqr instance that does have analysts